locopy/redshift.py

```diff
@@ -302,7 +312,10 @@ def load_and_copy(
 
         # copy files to S3
         s3_upload_list = self.upload_list_to_s3(upload_list, s3_bucket, s3_folder)
-        tmp_load_path = s3_upload_list[0].split(os.extsep)[0]
+        if os.path.isdir(local_file):
+            tmp_load_path = "/".join([s3_bucket, s3_folder])
```

Comment:
Might make sense to use `PathLib` or something else instead of `/` to ensure it is platform agnostic?